client/src/components/OverviewProfileCard.js

```diff
@@ -1,85 +1,112 @@
-import React, { useContext } from 'react';
-import { Card } from 'react-bootstrap';
-import styled from 'styled-components';
-import ViewSkills from '../components/ViewSkills';
-import StyledButton from '../constant/StyledButton';
-import avatar from '../assets/icons/avatar.svg';
-import { useHistory } from 'react-router-dom';
+import React, { useContext } from "react";
+import { Card } from "react-bootstrap";
+import styled from "styled-components";
+import ViewSkills from "../components/ViewSkills";
+import StyledButton from "../constant/StyledButton";
+import { useHistory } from "react-router-dom";
+import GitHubIcon from "./GitHubIcon.js";
+import LinkedinIcon from "./LinkedinIcon";
+import CvIcon from "./CvIcon";
+import EmailIcon from "./EmailIcon";
+import HiredLabel from '../constant/HiredLabel'
 
-
-const OverviewProfileCard = ({ profile, getProfile }) => {
-	let history = useHistory()
-
-	const handleClick = (name)=>{
-		getProfile(name)
-		history.push('/viewdetail')
-	};
-
-	return (
-		<CardContainer>
-			{/* <Img variant="top" src={profile.img||avatar} /> */}
-			<Img />
-			<CardBody>
-				<CardTitle>{profile.first_name} {profile.last_name}</CardTitle>
-				<CardText>
-				Dolorem ipsum quia dolor sit amet, excepturi sint occaecati
-				</CardText>
-				{/* <ViewSkills skills={profile.skills} /> */}
-				<StyledButton name='View Profile' handleClick={()=>handleClick(`${profile.user_name}`)} />
-			</CardBody>
-		</CardContainer>
-	);
+const OverviewProfileCard = ({ singleProfile, getProfile}) => {
+  let history = useHistory();
+  
+  const handleClick = async (id) => {
+    await getProfile(id);
+    history.push(`/profiles/${singleProfile.first_name}`);
+  };
+  
+  return (
+    <CardContainer>
+      {singleProfile.is_hired&&<HiredLabel/>}
+      <Img src={singleProfile.avatar_url}/>
+      <CardBody>
+        <CardTitle>
+          {singleProfile.first_name} {singleProfile.surname}
+        </CardTitle>
+        <SubContainer>
+          {singleProfile.about_me&&<CardText>{singleProfile.about_me}</CardText>}
+        </SubContainer>
+        <SubContainer>
+         {singleProfile.about_me&&<CardText>{singleProfile.interest}</CardText>}
+        </SubContainer>
+        <IconContainer>
+          {singleProfile.github_link&&<GitHubIcon gitHubLink={singleProfile.github_link}></GitHubIcon>}
+          {singleProfile.linkedin_link&&<LinkedinIcon linkedinLink={singleProfile.linkedin_link}></LinkedinIcon>}
+          <CvIcon CvLink={singleProfile.cv_link}></CvIcon>
+          <EmailIcon singleProfile={singleProfile}></EmailIcon>
+        </IconContainer>
+        <StyledButton
+          name="View Profile"
+          handleClick={() => handleClick(`${singleProfile.github_id}`)}
+        />
+      </CardBody>
+    </CardContainer>
```

Comment:
It's good practice to deconstruct your variables so that you don't have to repeat yourself.
e.g. `const { is_hired, avatar_url, etc...} = singleProfile;`